backend/src/test/java/de/janka/capstonedrazen/model/UserEntityTest.java

```diff
@@ -0,0 +1,77 @@
+package de.janka.capstonedrazen.model;
+
+import de.janka.capstonedrazen.SpringBootTests;
+import de.janka.capstonedrazen.repo.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import javax.annotation.Resource;
+import javax.transaction.Transactional;
+
+import java.awt.print.PrinterJob;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UserEntityTest extends SpringBootTests {
+
+    @AfterEach
```

Comment:
Vergiss nicht dass es auch eine @BeforeEach Annotation gibt. D.h. du kannst was in die Datenbank vor jedem Test schreiben, und dann die Datenbank diesbzgl testen, ist gelegentlich hilfreich und spart dir sehr viel Arbeit. Schau dir Matthias Tests durch, die sind meistens 1A